src/main/java/io/supertokens/webserver/api/mfa/DisableFactorAPI.java

```diff
@@ -0,0 +1,81 @@
+package io.supertokens.webserver.api.mfa;
+
+import com.google.gson.JsonObject;
+import io.supertokens.Main;
+import io.supertokens.TenantIdentifierWithStorageAndUserIdMapping;
+import io.supertokens.featureflag.exceptions.FeatureNotEnabledException;
+import io.supertokens.mfa.Mfa;
+import io.supertokens.pluginInterface.RECIPE_ID;
+import io.supertokens.pluginInterface.emailpassword.exceptions.UnknownUserIdException;
+import io.supertokens.pluginInterface.exceptions.StorageQueryException;
+import io.supertokens.pluginInterface.multitenancy.TenantIdentifierWithStorage;
+import io.supertokens.pluginInterface.multitenancy.exceptions.TenantOrAppNotFoundException;
+import io.supertokens.useridmapping.UserIdType;
+import io.supertokens.webserver.InputParser;
+import io.supertokens.webserver.WebserverAPI;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+public class DisableFactorAPI extends WebserverAPI {
+    private static final long serialVersionUID = -4641988458637882374L;
+
+    public DisableFactorAPI(Main main) {
+        super(main, RECIPE_ID.MFA.toString());
+    }
+
+    @Override
+    public String getPath() {
+        return "/recipe/mfa/factors/disable";
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
+        // API is tenant specific
+        JsonObject input = InputParser.parseJsonObjectOrThrowError(req);
+
+        String userId = InputParser.parseStringOrThrowError(input, "userId", false);
+        String factor = InputParser.parseStringOrThrowError(input, "factor", false);
+
```

Comment:
you need to lower case and trim (applies to all the apis)

Reply:
Only the factor should be trimmed/lowercased right? (coz we haven't done it in other recipes as well)

Reply:
If yes, done.